src/lints/trait_added_supertrait.ron

```diff
@@ -0,0 +1,58 @@
+SemverQuery(
+    id: "trait_added_supertrait",
+    human_readable_name: "non-sealed trait added new supertraits",
+    description: "A non-sealed trait added one or more supertraits, which breaks downstream implementations of the trait",
+    required_update: Major,
+    lint_level: Deny,
+    reference_link: Some("doc.rust-lang.org/cargo/reference/semver.html#generic-bounds-tighten"),
```

Comment:
```suggestion
    reference_link: Some("https://doc.rust-lang.org/cargo/reference/semver.html#generic-bounds-tighten"),
```

This is more likely to make it clickable in terminals

Reply:
Yes, we definitely want the `https://` in front. In fact, it would be great to have a test for this, since we don't have this policy written down anywhere.